docs/code/FilterParameters.py

```diff
@@ -11,15 +11,13 @@
 #
```

Comment:
Are these empty comment lines required for something? If not, please remove them.

Reply:
They are for there for readability.